alacritty.yml

```diff
@@ -454,9 +471,36 @@
 #   - ToggleFullscreen
 #   - SpawnNewInstance
 #   - ClearLogNotice
+#   - ClearSelection
 #   - ReceiveChar
 #   - None
 #
+#   (`mode: Vi` only):
+#   - Open
+#   - Up
+#   - Down
+#   - Left
+#   - Right
+#   - First
+#   - Last
+#   - FirstOccupied
+#   - High
+#   - Middle
+#   - Low
+#   - SemanticLeft
+#   - SemanticRight
+#   - SemanticLeftEnd
+#   - SemanticRightEnd
+#   - WordRight
+#   - WordLeft
+#   - WordRightEnd
+#   - WordLeftEnd
+#   - Bracket
+#   - ToggleNormalSelection
+#   - ToggleLineSelection
+#   - ToggleBlockSelection
+#   - ToggleSemanticSelection
```

Comment:
I believe this is a bug...

The `SemanticRight` action will move over the whole string `12:30` while selections in this mode will only move over the `12`. It seems the semantic selection mode is currently a word selection mode.

I still find the inclusion of semantic selection mode together with semantic movements a bit odd, but at the very least they should be consistent with each other. Keeping the code clean in this respect is paramount, to avoid issues like this.

Reply:
My bad, I got confused because I expected semantic to be vi's `WORD` but in fact our `Word` is.

So the motions are ordered like `Normal` < `Semantic` (akin to vi's `word`) < `Word` (akin to vi's `WORD`).